sslyze/plugins/http_headers_plugin.py

```diff
@@ -84,7 +85,7 @@ def _get_security_headers(cls, server_info):
         # "If you are serving an additional redirect from your HTTPS site, that redirect must still have the HSTS
         # header (rather than the page it redirects to)."
 
-        return hsts_header, hpkp_header, hpkp_report_only, certificate_chain
+        return hsts_header, hpkp_header, hpkp_report_only, certificate_chain, expect_ct_header
```

Comment:
Can you put the `expect_ct_header` return value right after `hpkp_header` (for consistency)?